### PR TITLE
Managing seats article

### PR DESCRIPTION
**Pull request by sbastn**
This PR adds a new article explaining how to manage seats.

Preview: https://deploy-preview-1080--dnsimple-support.netlify.app/articles/managing-seats/

### CODE REVIEW

File: content/articles/managing-seats.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# Managing seats
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+## Overview
+
+Seats and Team members are only available in our Teams or Enterprise plan. The Team plan includes an initial allocation of 3 seats at no additional cost. These seats allow you to have up to 3 members actively involved in domain management. Any team members beyond the initial 3 will incur extra costs. [See our pricing page for more details](https://dnsimple.com/pricing)
```
Comment (jacegu) [inline content/articles/managing-seats.markdown:17]:
```suggestion
Seats and Team members are only available in our Teams or Enterprise plan. The Teams plan includes an initial allocation of 3 seats at no additional cost. These seats allow you to have up to 3 members actively involved in domain management. Any team members beyond the initial 3 will incur extra costs. [See our pricing page for more details](https://dnsimple.com/pricing)
```

**Review by jacegu — APPROVED** [review]

**Comment by jacegu** [comment]
@sbastn do you think this article could use a couple screenshots?

**Review by itsalyse — APPROVED** [review]
copy lgtm 👍 I do think a couple screenshots, especially under the Decreasing and Inviting sections, would be useful here

**Comment by sbastn** [comment]
Thanks for the feedback! I've added screenshots. You can access the preview link in the summary of the PR.

**Pull request merged**